sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -260,6 +260,10 @@ def recognize_pii_entities(  # type: ignore
             be used for scoring, e.g. "latest", "2019-10-01". If a model-version
             is not specified, the API will default to the latest, non-preview version.
         :keyword bool show_stats: If set to true, response will contain document level statistics.
+        :keyword domain_filter: Filters the response entities to ones only included in your specified domain.
+            I.e., if set to 'PHI', will only return entities in the Protected Healthcare Information domain.
+            Currently only supports 'PHI'. See https://aka.ms/tanerpii for more information.
```

Comment:
Not sure there is value adding the support. By looking at the enum and documentation, people can see that. Otherwise next time they add something someone will have to change this comment.

Reply:
That's fair, I can ermove that and shorten the comment a bit